packages/ui/src/coreModules/user/selectors.js

```diff
@@ -13,7 +13,11 @@ export const getUser = state => {
   return state.user
 }
 
-export const getUserName = state => {
```

Comment:
this line has to be moved to https://github.com/DINA-Web/dina-collections/pull/324/commits/339a6f403a949467f048beeaffd8d7c0089acb8b